style.md

```diff
@@ -545,6 +546,102 @@ const (
 // LogToStdout=0, LogToFile=1, LogToRemote=2
 ```
 
+### Use time
+
+Time is complicated. Incorrect assumptions often made about time include the
+following.
+
+1. A day has 24 hours
+2. An hour has 60 minutes
+3. A week has 7 days
+4. A year has 365 days
+5. [And a lot more](https://infiniteundo.com/post/25326999628/falsehoods-programmers-believe-about-time)
```

Comment:
(just to confirm, go's time package does handle the cases above correctly right?)

Reply:
Yeah, it handles them correctly -- mostly. It doesn't support leap seconds per the documentation (see also golang/go#15247, golang/go#8728).

> The calendrical calculations always assume a Gregorian calendar, with no leap seconds.

Reply:
Ah, do you think we should make a note of that here?

Reply:
Yeah, I'll add a thing.